test/NodeGuard.Tests/Helpers/WalletParserTests.cs

```diff
@@ -149,4 +150,315 @@ public void ParseOutputDescriptor_InvalidSegwitSingleSig(string descriptor, stri
 
     }
     
+    // Testing GetOutputDescriptor method
+    [Fact]
+    public void GetOutputDescriptor_NativeSegwits()
```

Comment:
Please if you can order the tests as Arrange/Act/Assert

Reply:
Done! 🏁